swift_browser_ui_frontend/src/common/globalFunctions.js

```diff
@@ -5,4 +5,9 @@ export function toggleCreateFolderModal(folderName) {
   if (folderName) {
     store.commit("setFolderName", folderName);
   }
-} 
\ No newline at end of file
+}
+
+export function modifyBrowserPageStyles() {
+  const element = document.getElementById("subContainer");
+  element.classList.toggle("subContainer-additionalStyles");
+}
```

Comment:
```suggestion
}
```
Missing ending newline, not sure if suggestion actually adds it though.